### PR TITLE
fixes round end sounds and adds a new one thanks to BlackPantsLegion Tex

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: GuyonBroadway
add: Added a new roundend sound voice by Goonstations Bill and taken from a Black pants Legion Colab video.
fix: fixed roundend sounds.
/:cl:

I was watching one of Tex's videos and there was a voice snippet in there that tickled me greatly, in my efforts to make it a round-end sound I fixed round end sounds not working.

This sound snippet is used with permission.

image

Video here

https://youtu.be/k2rH6oiBglU

### CODE REVIEW

**Comment by JohnGinnane** [comment]
what's wrong with roundend sounds? They were working fine at the weekend as far as I know

**Comment by GuyonBroadway** [comment]
Were they? I might be crazy but I've not heard any in a while, in any case we needed to mirror some changes to the playing roundend sound proc which is what I did here.

**Comment by JohnGinnane** [comment]
Oooh okay, I might rework the roundend sound thing on /tg/ to use something like the lobby sounds, that way we don't need to override this entire proc

**Comment by GuyonBroadway** [comment]
that would be nice.

File: hippiestation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -1,4 +1,6 @@
 /datum/controller/subsystem/ticker/Shutdown()
+	gather_newscaster() //called here so we ensure the log is created even upon admin reboot
```
Comment (Carbonhell) [inline hippiestation/code/controllers/subsystem/ticker.dm:2]: what's this?

**Comment by Carbonhell** [comment]
sure +1

File: hippiestation/code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (GuyonBroadway) [reply hippiestation/code/controllers/subsystem/ticker.dm:2]: Some newscaster persistence system tg added, I think In any case its something we forgot to mirror when tg updated the ticker/shutdown().

**Review by praisenarsie — APPROVED** [review]

**Pull request merged**